### ISSUE TITLE
Throws on multiple matched selectors

### ISSUE THREAD

**Issue by pocketjoso** [bevacqua/apartment#1]
Unfortunately apartment throws on certain css, breaking latest version of Penthouse :/

Minimal test case:

```
apartment(`
::-moz-selection {
  color: #fff;
}
::selection {
  color: #fff;
}`, {
  selectors: [
    '::(-moz-)?selection'
  ]
}
```

The thrown error

```
/Users/jso/penthouse/node_modules/apartment/src/apartment.js:32
    if (inspected.type === 'rule') {
                 ^

TypeError: Cannot read property 'type' of undefined
```

Cheers

**Comment by pocketjoso** [comment]
Haven't had time to understand your code completely to debug,
but it seems like [these lines](https://github.com/bevacqua/apartment/blob/master/src/apartment.js#L31-L33) accidentally grow your scope from the rule you're currently treating to.. everything? Which here causes a future rule to be deleted, hence why it's missing later..

**Comment by bevacqua** [comment]
Fixed in `1.1.1`, added test cases. Let me know if this fixes your use case

**Comment by pocketjoso** [comment]
Awesome, thanks for the quick response! Will update penthouse now. :)
